components/ui/SplitSectionText.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <div>
+    <h2
+      class="
+        prose prose-sm
```

Comment:
Incorrect use of 'prose' class here. prose class is  meant for markdown file content. See: https://github.com/tailwindlabs/tailwindcss-typography